apps/web/src/modules/create-proposal/constants/transactionType.tsx

```diff
@@ -40,6 +41,12 @@ export const TRANSACTION_TYPES = {
     icon: 'airdrop',
     iconBackdrop: 'rgba(28, 182, 135, 0.1)',
   },
+  [TransactionType.ESCROW]: {
+    title: 'Escrow Milestones',
```

Comment:
Is "Escrow Milestones" the agreed upon name for this ? It is fine. But maybe we can do better?

Reply:
Yeah as per figma ig